vspace/src/main/java/edu/asu/diging/vspace/web/staff/ExhibitionConfigurationController.java

```diff
@@ -77,19 +81,26 @@ public String showExhibitions(Model model) {
      * @param spaceParam
      * @param attributes
      * @return
-     */
+     */    
     @RequestMapping(value = "/staff/exhibit/config", method = RequestMethod.POST)
     public RedirectView createOrUpdateExhibition(HttpServletRequest request,
             @RequestParam(required = false, name = "exhibitionParam") String exhibitID,
-            @RequestParam("spaceParam") String spaceID, @RequestParam("title") String title,
-            @RequestParam("exhibitMode") ExhibitionModes exhibitMode,
-            @RequestParam(value = "customMessage", required = false, defaultValue = "") String customMessage,
-            @RequestParam("exhibitLanguage") List<String> languages,
-            @RequestParam("defaultExhibitLanguage") String defaultLanguage,
+            @RequestParam("spaceParam") String spaceID,
+            @Valid @ModelAttribute("exhibitionConfigurationForm") ExhibitionConfigurationForm exhibitionConfigForm,
+            BindingResult result,           
             RedirectAttributes attributes) throws IOException {
-
-        ISpace startSpace = spaceManager.getSpace(spaceID);
-
+        if(result.hasErrors()) {
+            attributes.addAttribute("showAlert", true);
+            attributes.addAttribute("alertType", "danger");
+            attributes.addAttribute("message", result.getFieldError().getDefaultMessage());
+            return new RedirectView(request.getContextPath() + "/staff/exhibit/config");
+        }
+        String title = exhibitionConfigForm.getTitle();
+        ExhibitionModes exhibitMode = exhibitionConfigForm.getExhibitionMode();
+        List<String> languages = exhibitionConfigForm.getExhibitLanguage();
+        String defaultLanguage = exhibitionConfigForm.getDefaultExhibitLanguage();
+        String customMessage = exhibitionConfigForm.getCustomMessage();
+        ISpace startSpace = spaceManager.getSpace(spaceID);       
```

Comment:
If these are not use more than once, just merge these links with where they are used, e.g.
`exhibition.setStartSpace(paceManager.getSpace(spaceID));`